vignettes/ggPMX-ggplot.Rmd

```diff
@@ -0,0 +1,81 @@
+---
+title: "How to use ggplot2 syntax with ggPMX"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{How to use ggplot syntax with ggPMX}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+We will create a simple ggPMX controller using the theophylline example:
+
+```{r, warning=FALSE, message=FALSE}
+
+library(ggPMX)
+library(ggplot2)
+
+ctr <- theophylline()
+```
+
+Each ggPMX plot is an object with class ggplot:
+
+```{r, warning=FALSE, message=FALSE}
+p <- ctr %>% pmx_plot_dv_ipred()
+class(p)
+```
+
+
+Therefore we can use usual ggplot2 syntax to customize a ggPMX plot.
+
+Let's look at some examples.
+
+1. Facet
+
+ggPMX :
+
+```{r fig.height=5, fig.width=8, warning=FALSE, message=FALSE}
+ctr %>% pmx_plot_npde_pred(strat.facet= ~SEX)
+```
+
+ggplot:
+
+```{r fig.height=5, fig.width=8, warning=FALSE, message=FALSE}
+ctr %>% pmx_plot_npde_pred() + facet_wrap(~SEX)
+```
```

Comment:
Perhaps a comment here about remembering that `ggplot2` does not use the `%>%` operator, since that is a common mistake for beginners.

Reply:
Okay, will add the note